tests/results/dadc624d4454e10293dbd1b701b9ee9f99ef83b4cd07b695111d37eb95abcff8_archive_from_javascript/params.json

```diff
@@ -0,0 +1,19 @@
+{
+    "task_depth": 1,
+    "temp_submission_data": {
+        "ancestry": [
+            [
+                {
+                    "sha256": "5ff5b215aca17548a545676c6d0ab701071976f76632b4b6b2fe25dfc29d6fe9",
+                    "type": "code/javascript",
+                    "parent_relation": "DROPPED"
```

Comment:
Is DROPPED a real option?
I thought it would only ever be one of [those](https://github.com/CybercentreCanada/assemblyline-v4-service/blob/v4.5.0.stable54/assemblyline_v4_service/common/task.py#L20)?